### PR TITLE
Fix error in boost, and added some examples

### PR DESCRIPTION
**Pull request by angoyd**

### CODE REVIEW

**Comment by angoyd** [comment]
I will continue to test the other functions.

**Comment by LaStrada** [comment]
Thanks for the PR! Will look into this PR (and in general this repo) in a few days. Summer = vacation time...

File: examples/scanner.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+        DefaultDelegate.__init__(self)
+
+    def handleDiscovery(self, dev, isNewDev, isNewData):
+        if isNewDev:
+            print ("Discovered device", dev.addr)
+        elif isNewData:
+            print ("Received new data from", dev.addr)
+
+scanner = Scanner().withDelegate(ScanDelegate())
+devices = scanner.scan(10.0)
```
Comment (LaStrada) [inline examples/scanner.py:15]: Shouldn't there be a simple filter or something to remove most of the devices? My device's name is `Intellivent Sky`.

File: examples/sky-set-boost-2min.py [hunk 2]
```diff
@@ -0,0 +1,3 @@
+from pyfreshintellivent.pyfreshintellivent.FreshIntellivent import Sky
+p = Sky("Fan adress","authcode from sky-auth-ex",False)
```
Comment (LaStrada) [inline examples/sky-set-boost-2min.py]: Don't need to include `False` + spelling.
```suggestion
p = Sky("Fan address","authcode from sky-auth-ex")
```

File: examples/sky-set-boost-off.py [hunk 3]
```diff
@@ -0,0 +1,3 @@
+from pyfreshintellivent.pyfreshintellivent.FreshIntellivent import Sky
+p = Sky("Fan adress","authcode from sky-auth-ex",False)
```
Comment (LaStrada) [inline examples/sky-set-boost-off.py]: Don't need to include `False`.
```suggestion
p = Sky("Fan address","authcode from sky-auth-ex", False)
```

File: examples/sky-get-info-ex.py [hunk 4]
```diff
@@ -0,0 +1,10 @@
+from pyfreshintellivent.pyfreshintellivent.FreshIntellivent import Sky
+p = Sky("Fan adress","authcode from sky-auth-ex",False)
```
Comment (LaStrada) [inline examples/sky-get-info-ex.py]: Don't need to include False + spelling.
```suggestion
p = Sky("Fan address", "authcode from sky-auth-ex")
```

**Review by LaStrada — CHANGES_REQUESTED** [review]
Need to change a few things. Also merged in a linting PR, so a few linting issues (and conflicts) have to be addressed.

Edit: And again, thanks for the PR! Great changes, and I think your changes to the code itself fixes the boost command.

**Comment by LaStrada** [comment]
Maybe a bit bad timing... But I looked into the changes made in this PR, and I started a new branch yesterday. I had to rewrite the whole project to use `bleak` instead of `bluepy`, because I want it to run on macOS, and also make the transition to a Home Assistant integration easier.

File: examples/scanner.py (hunk 1 above)
Comment (angoyd) [reply examples/scanner.py:15]: Would probably be nice maby some more tests before applying filter ?

**Comment by angoyd** [comment]
Ahh lets skip this then I will merge from your repo and test there.

**Pull request closed without merging**